src/factom-event-emitter.js

```diff
@@ -321,6 +437,28 @@ class FactomEventEmitter extends EventEmitter {
             }
         });
     }
+
+    // Emit new pending factoid transactions for user-defined FCT address
+    _emitPendingFactoidTransaction(fctAddress, transactions) {
+        this.emit(`newPendingTransaction:${fctAddress}`, transactions);
```

Comment:
nit: should the hardcoded string be replaced by the variable `PENDING_EVENT.newPendingTransaction`?